### PR TITLE
Add VideoNote

### PR DESCRIPTION
**Pull request by bomjacob**
refs #617 
This is where it gets weird....
According to telegram length is Video width and height.. but how that works with one parameter I couldn't tell you.. it also seems to be required but is not marked as such in the documentation....

### CODE REVIEW

File: telegram/bot.py [hunk 1]
```diff
@@ -672,6 +672,66 @@ def send_voice(self,
...
+                        reply_to_message_id=None,
+                        reply_markup=None,
+                        timeout=20.,
+                        **kwargs):
+        """As of v.4.0, Telegram clients support rounded square mp4 videos of up to 1 minute
+        long. Use this method to send video messages
+
+        Args:
+            chat_id (int|str): Unique identifier for the message recipient - Chat id.
+            voice: Video note to send. Pass a file_id as String to send a video note that exists
```
Comment (tsnoam) [inline telegram/bot.py]: s/voice/vide_note/
type: InputFile|str

File: telegram/bot.py [hunk 2]
```diff
@@ -672,6 +672,66 @@ def send_voice(self,
...
+
+        Raises:
+            :class:`telegram.TelegramError`
+
+        """
+        url = '{0}/sendVideoNote'.format(self.base_url)
+
+        data = {'chat_id': chat_id, 'video_note': video_note}
+
+        if duration:
```
Comment (tsnoam) [inline telegram/bot.py]: if duration `is not None`
(and same for `length` below)

File: telegram/inputfile.py [hunk 3]
```diff
@@ -66,6 +67,9 @@ def __init__(self, data):
         elif 'certificate' in data:
             self.input_name = 'certificate'
             self.input_file = data.pop('certificate')
+        elif 'video_note' in data:
+            self.input_name = 'video_note'
+            self.input_file = data.pop('video_note')
```
Comment (tsnoam) [inline telegram/inputfile.py]: I wonder - we have a lot of branching here. Should we move to a dict here? It's much more pythonic.

File: telegram/inputfile.py [hunk 4]
```diff
@@ -117,7 +121,7 @@ def to_form(self):
                 form_boundary, 'Content-Disposition: form-data; name="%s"' % name, '', str(value)
             ])
 
-        # Add input_file to upload
+# Add input_file to upload
```
Comment (tsnoam) [inline telegram/inputfile.py]: Please return the leading indentation

File: telegram/message.py [hunk 5]
```diff
@@ -125,6 +129,7 @@ def __init__(self,
                  sticker=None,
                  video=None,
                  voice=None,
+                 video_note=None,
```
Comment (tsnoam) [inline telegram/message.py]: Same note like I left on #628 - should new parameters be added in the middle or in the end of the function?

File: telegram/inputfile.py (hunk 4 above)
Comment (bomjacob) [inline telegram/inputfile.py]: Right, danm yapf

File: telegram/bot.py (hunk 2 above)
Comment (bomjacob) [inline telegram/bot.py]: Sure? We don't do that anywhere else in bot.py?

File: telegram/inputfile.py (hunk 3 above)
Comment (bomjacob) [inline telegram/inputfile.py]: Not sure what you meant.. does this work?

**Pull request merged**